test/e2e/test_cap_kill.py

```diff
@@ -140,3 +142,177 @@ def test_cap_kill_not_inherited_by_running_jobs(
         logs_client=logs_client,
         expected_pattern=possible_error_message_re_pattern,
     )
+
+
+@pytest.mark.skipif(
+    os.environ["OPERATING_SYSTEM"] == "windows",
```

Comment:
nit: just a bit more future-proof if/when we support other operating systems

```suggestion
    os.environ["OPERATING_SYSTEM"] != "linux
```